engine/link/link.asm

```diff
@@ -1571,10 +1571,10 @@ Unreferenced_Function28b42:
 	ld bc, SCREEN_WIDTH - 2
 	call ByteFill
 	hlcoord 2, 16
-	ld de, .Cancel
+	ld de, .cancel_string
 	jp PlaceString
 
-.Cancel:
+.cancel_string:
```

Comment:
As per https://github.com/pret/pokecrystal/blob/master/STYLE.md
This should be `CancelString`

Reply:
Yup, I definitely misread the style guide.  I've fixed that and the other similar issues in the other edits.